test/spix/conftest.py

```diff
@@ -72,10 +75,13 @@ def process():
         "./output/bin/qfield.app/qfield_spix.exe",
         "./output/bin/qfield.app/Contents/MacOS/qfield_spix",
     ]
+
+    projectpath = str(Path(__file__).parent.parent / "testdata" / "test_wms.qgz")
```

Comment:
Could we parametrize this? Something like https://stackoverflow.com/a/33879151/2319028 so we could have an array of projects to test?